Broken build due to linting errors

Hey @beaufortfrancois, #351 broke the build due to linting errors:

```sh
/home/travis/build/GoogleChrome/samples/web-bluetooth/characteristic-properties.js
  4:19  error  Missing radix parameter  radix
  9:26  error  Missing radix parameter  radix
/home/travis/build/GoogleChrome/samples/web-bluetooth/device-disconnect.js
  24:21  error  Missing radix parameter  radix
/home/travis/build/GoogleChrome/samples/web-bluetooth/device-info.js
  6:21  error  Missing radix parameter  radix
/home/travis/build/GoogleChrome/samples/web-bluetooth/get-characteristics.js
  4:19  error  Missing radix parameter  radix
  9:26  error  Missing radix parameter  radix
/home/travis/build/GoogleChrome/samples/web-bluetooth/notifications.js
   6:19  error  Missing radix parameter  radix
  11:26  error  Missing radix parameter  radix
✖ 8 problems (8 errors, 0 warnings)
```

Could you please disable that rule since the missing radix was intentional? Or re-add the radix?

(And in general, please make sure the build is green before merging PRs with a TBR?)

--- a/web-bluetooth/.eslintrc
+++ b/web-bluetooth/.eslintrc
@@ -4,6 +4,7 @@
     "log": true
   },
   "rules": {
+    "radix": 0,
     "max-len": 0,
     "no-unused-vars": 0,
     "no-undef": 0

